.github/ISSUE_TEMPLATE/BUG_BACKEND.md

```diff
@@ -0,0 +1,35 @@
+---
+name: ⚠️ Report Backend Bug
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[backend] <Bug Name>"
+labels: kind/bug, area/backend
+---
+
+### Environment
+
+*  How did you deploy Kubeflow Pipelines (KFP)?
+<!-- See [an introduction of all options](https://www.kubeflow.org/docs/pipelines/installation/overview/). -->
```

Comment:
Suggested revision, please replace all instances:

For more information, see an [overview of KFP installation options](https://www.kubeflow.org/docs/pipelines/installation/overview/).

Reply:
Sounds good! Done.